mne_bids/tests/test_write.py

```diff
@@ -480,6 +567,14 @@ def test_edf(_bids_validate):
     write_raw_bids(raw, bids_basename, output_path)
     _bids_validate(output_path)
 
+    # test anonymize and convert
+    raw = mne.io.read_raw_edf(raw_fname, preload=True)
+    raw.preload = False
```

Comment:
why do you need to hack this?

Reply:
I'm not sure, that was there before, I just copied. Here is are the lines earlier in that test function which comment explains (or not) it:

raw = mne.io.read_raw_edf(raw_fname, preload=True)
# XXX: hack that should be fixed later. Annotation reading is
# broken for this file with preload=False and read_annotations_edf
raw.preload = False

Reply:
hum... is it still the case with 0.19?

Reply:
Maybe @jasmainak or @teonbrooks knows?

Reply:
I think this has been resolved with all the landed annotation features. maybe we should try bumping the mne required version and remove the preload

Reply:
Try @alexrockhill :) I don't know. I remember this was a bit involved because of all the different kinds of annotations you can have in EDF files.

Reply:
what was the conclusion here?

Reply:
Just to follow up it was fine with the normal preload = False